### PR TITLE
Feat/repo dx improvement

### PR DESCRIPTION
**Pull request by felipeegasneris**
mod lerna config,  add commitizen to improve DX, and other stuff 👯

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,21 +1,59 @@
...
+    "commitizen": "4.2.3",
+    "cz-conventional-changelog": "^3.3.0",
+    "husky": "^4.3.8",
+    "lint-staged": "^10.5.3"
   },
   "private": true,
   "scripts": {
-    "bootstrap": "lerna bootstrap",
+    "lerna:build": "lerna run build",
+    "lerna:build:current": "npm run lerna:build --since origin/master --include-filtered-dependents --include-filtered-dependencies",
```
Comment (sandinosaso) [inline package.json:19]: What does the --include-filtered-dependents --include-filtered-dependencies does? How it helps us?

File: package.json [hunk 2]
```diff
@@ -1,21 +1,59 @@
...
+    "lerna:lint:current": "yarn run lerna:lint --since origin/master --include-filtered-dependents --include-filtered-dependencies",
+    "lerna:updated": "lerna updated",
+    "lerna:unit": "lerna run unit --concurrency=1",
+    "lerna:unit:current": "yarn run lerna:unit --since origin/master --include-filtered-dependents --include-filtered-dependencies",
+    "lerna:test": "lerna run test --concurrency=1",
+    "lerna:test:current": "yarn run lerna:test --since origin/master --include-filtered-dependents --include-filtered-dependencies",
+    "lerna:ls": "lerna ls",
+    "lerna:ls:current": "yarn run lerna:ls --since origin/master --include-filtered-dependents --include-filtered-dependencies",
+    "lerna:bootstrap": "lerna bootstrap",
+    "lerna:publish": "npm run lerna:build && lerna publish --c --yes",
```
Comment (sandinosaso) [inline package.json:31]: We are using more `yarn`, maybe we could have all commands to use yarn?

File: package.json [hunk 3]
```diff
@@ -1,21 +1,59 @@
...
   "private": true,
   "scripts": {
-    "bootstrap": "lerna bootstrap",
+    "lerna:build": "lerna run build",
+    "lerna:build:current": "npm run lerna:build --since origin/master --include-filtered-dependents --include-filtered-dependencies",
+    "lerna:install": "lerna run install",
+    "lerna:lint": "lerna run lint",
+    "lerna:lint:current": "yarn run lerna:lint --since origin/master --include-filtered-dependents --include-filtered-dependencies",
+    "lerna:updated": "lerna updated",
+    "lerna:unit": "lerna run unit --concurrency=1",
```
Comment (sandinosaso) [inline package.json:24]: What does lerna run unit do? If there is no unit script 🤔

Comment (felipeegasneris) [reply package.json:24]: nothing, for now 😄  xd

File: package.json (hunk 2 above)
Comment (felipeegasneris) [reply package.json:31]: yeah sure!

**Comment by sandinosaso** [comment]
@felipeegasneris Can you fetch develop updated, and rebase this branch on top of developing: I did a hotfix that fixed the failed test that is making this PR fail (probably before you changed this to use Lerna correctly, that package was pushed with a failed test without us noticing it, now you exposed that problem :) and I am glad you did it)

**Review by s0wk — APPROVED** [review]

**Review by sandinosaso — APPROVED** [review]
Thank you for explaining the commands to me :) :shipit:

**Pull request merged**